test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/DeleteTests.cs

```diff
@@ -73,6 +73,19 @@ async Task<FhirClientException> ExecuteAndValidateGoneStatus(Func<Task> action)
             }
         }
 
+        [Fact]
+        public async Task GivenADeletedResource_WhenSearchWithNotModifier_ThenDeletedResourcesShouldNotBeReturned()
```

Comment:
Is there a way to write this test using tags? I don't like relying on us never using the Flag resource type in the future, it looks like it could cause hard to debug failures down the road.

Reply:
I did think about that, but I can't use the tag value later to search, because the resources will be deleted, and the tags should also be deleted at that point.